cibuild.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+WD=$PWD
+nuttx=$WD/../nuttx
+apps=$WD/../apps
+tools=$WD/../tools
+prebuilt=$WD/../prebuilt
+
+# genromfs
+install=("if [ ! -d \"$prebuilt/genromfs/usr/bin\" ]; then \
+            if [ ! -d "$tools" ]; then \
+              git clone https://github.com/nuttx/tools.git $tools; \
+            fi; \
+            mkdir -p $prebuilt; cd $tools; \
+            tar zxf genromfs-0.5.2.tar.gz -C $prebuilt; \
+            cd $prebuilt/genromfs-0.5.2; \
+            make install PREFIX=$prebuilt/genromfs; \
+            cd $prebuilt; rm -rf genromfs-0.5.2; \
+          fi; \
+          genromfs -h; \
+          ")
+path=("$prebuilt/genromfs/usr/bin")
+
+# kconfig-frontends
+install+=("if [ ! -d \"$prebuilt/kconfig-frontends/bin\" ]; then \
+             cd $tools/kconfig-frontends; \
+             ./configure --prefix=$prebuilt/kconfig-frontends --enable-mconf --disable-gconf --disable-qconf --enable-static; \
+             make install; \
```

Comment:
could this also benefit of the PREFIX parameter?

Reply:
same as above.

Reply:
ok